ui/raidboss/data/04-sb/dungeon/ala_mhigo.js

```diff
@@ -0,0 +1,206 @@
+'use strict';
+
+[{
+  zoneRegex: /Ala Mhigo/,
+  timelineFile: 'ala_mhigo.txt',
+  timelineTriggers: [
+    {
+      id: 'Ala Mhigo Umoving Troika',
+      regex: /Unmoving Troika/,
+      beforeSeconds: 5,
+      alertText: function(data) {
+        if (data.role == 'tank') {
+          return {
+            en: 'Tank cleave on YOU',
+          };
+        }
+        return {
+          en: 'Avoid tank cleave',
+        };
+      },
+    },
+  ],
+  triggers: [
+    {
+      id: 'Ala Mhigo Electromagnetic Field',
+      regex: / 14:204D:Magitek Scorpion starts using Electromagnetic Field/,
+      condition: function(data) {
+        return data.role == 'healer';
+      },
+      infoText: {
+        en: 'AoE',
+      },
+    },
+    {
+      id: 'Ala Mhigo Mana Burst',
+      regex: / 14:204F:Aulus [mM]al Asina starts using Mana Burst/,
```

Comment:
regexes are now case insensitive, so this isn't necessary anymore.  :)

Reply:
Oh, did I miss that when it got merged? I don't remember seeing it specifically, but I might have missed it in the whirlwind panicstevenson submitted recently. Really awesome though, that's going to make things a LOT easier.